pkg/controllers/status/cluster_status_controller.go

```diff
@@ -368,7 +352,9 @@ func getKubernetesVersion(clusterClient *util.ClusterClient) (string, error) {
 func getAPIEnablements(clusterClient *util.ClusterClient) ([]clusterv1alpha1.APIEnablement, error) {
 	_, apiResourceList, err := clusterClient.KubeClient.Discovery().ServerGroupsAndResources()
 	if err != nil {
-		return nil, err
+		// if we fail, just log it.
+		// In case discovery client return part of succeed groups: https://github.com/kubernetes/kubernetes/blob/33de444861d3de783a6618be9d10fa84da1c11b4/staging/src/k8s.io/client-go/discovery/discovery_client.go#L257
+		klog.Errorf("unable to retrieve the complete list of server APIs: %v", err)
```

Comment:
May have effects on scheduler `APIInstalled` plugin.

Reply:
The `err` will be returned eventually, which means the `incompleted` list will be discarded anyway.
No idea how to handle this case yet.